platforms/ide/tooling-api/src/main/java/org/gradle/tooling/events/problems/internal/DefaultExceptionContainer.java

```diff
@@ -16,19 +16,20 @@
 
 package org.gradle.tooling.events.problems.internal;
 
+import org.gradle.tooling.Failure;
 import org.gradle.tooling.events.problems.ExceptionContainer;
 
 import javax.annotation.Nullable;
 
 public class DefaultExceptionContainer implements ExceptionContainer {
```

Comment:
🤔 The name of these container types should probably also change to `FailureContainer`.